pkg/arvo/app/file-server.hoon

```diff
@@ -75,8 +70,12 @@
     ?-  -.act
         %serve-dir
       =*  url-base  url-base.act
-      ?:  (~(has by serving) url-base)
-        ~|("url already bound to {<(~(got by serving) url-base.act)>}" !!)
+      =/  served  (~(get by serving) url-base)
+      ?^  served
+        ?.  =(clay-base.act clay-base.u.served)
+          ~|("url already bound to {<(~(got by serving) url-base.act)>}" !!)
+        :-  [%pass url-base %arvo %e %connect [~ url-base] %file-server]~
+        this(serving (~(put by serving) url-base [clay-base.act public.act]))
       :-  [%pass url-base %arvo %e %connect [~ url-base] %file-server]~
       this(serving (~(put by serving) url-base [clay-base.act public.act]))
```

Comment:
Being able to write this as a single `?:` aside, this now lets you double-connect on the same `url-base` as long as you're serving different files? But forbids you from serving the same files in multiple locations? The `%connect` here will fail if it's already bound by something...

None of that seems very sane to me. Is this intentional? Can you explain?

(Just realized you can actually scry into eyre to find out if there's a binding in place, but it's not the best interface. I'll try and put something nicer there soon, would be nice to be able to do checks on that.)

Reply:
(idk why it says "outdated" on this, I can still see the quoted code in the diff...)

Reply:
Fixed the issue this solved via an alternative means (making +on-load arms stateful everywhere), reverted back to prior implementation.